src/resources/views/columns/image.blade.php

```diff
@@ -0,0 +1,18 @@
+<td>
+  @if( !empty($entry->{$column['src']}) )
+    <a
+      href="{{ $entry->{$column['src']} }}"
+      target="_blank"
+    >
+      <img
+        src="{{ $entry->{$column['src']} }}"
```

Comment:
This is a bit atypical, but I understand the rationale behind it. 

Usually columns use ```$entry->{$column['name']}``` for the information, not ```src```. And by that I mean the developer probably expects to define the column like this:
```diff
[
   'name' => 'profile_image', // The db column name
-   'src' => 'profile_image',
   'label' => "Profile Image", // Table column heading
   'type' => 'image'
]
```

1) Using "src" in addition to "name" is a bit atypical and I think, not needed. But I might be wrong, what do you think? Can I rename ```src``` into ```name```?

2) Another problem I see is if people do not store the full path in the database. What i mean is that this will work perfectly for external links, but for internal links, where in the db column you only have ```profile_images/81hr891hd918.jpg``` this field will not work. And I think this is the most common use case. The beauty of Laravel is that it has some intuitive fallbacks. I just tested the ```asset()``` helper, and it looks like [if you pass an URL to it, it does nothing. If you pass a regular string, it turns it into an URL](https://cl.ly/151I1B1K3X2X). So I think just using ```{{ asset($entry->{$column['name']}) }}``` both here and above in the href will make, this work for more use cases.

What do you think? Good changes? Should I go ahead and do them or do you have time to push the changes and see if they work with your use cases too?

Reply:
1) I used name first, then changed it to src. There is no big deal moving it back.
2) Makes total sense. I was thinking it as a field to pass any kind of src, including base64, but the url or base64 image will mostly come from the DB. I overcomplicated it more than needed. 

Will be making this changes and updating the PR  as soon as posible.